TheraLang.Tests/Services/MemberFeeServiceTests.cs

```diff
@@ -111,7 +111,23 @@ public void AddAsync_ShouldThrowException()
             mockUnitOfWork.Verify(x => x.SaveChangesAsync(), Times.Never());
             mockRepo.Verify(x => x.Add(It.IsAny<MemberFee>()), Times.Never());
             act.Should().Throw<NullReferenceException>()
-                .WithMessage($"{nameof(MemberFeeDto)} cannot be null");
+                .WithMessage("Object reference not set to an instance of an object.");
+        }
+        [Fact]
+        public void AddAsync_ShouldThrowInvalidArgumentException()
+        {
+            mockUnitOfWork.Setup(x => x.SaveChangesAsync()).Verifiable();
+            mockRepo.Setup(x => x.Add(It.IsAny<MemberFee>())).Verifiable();
+
+            MemberFeeService memberFeeService = new MemberFeeService(mockUnitOfWork.Object);
+
+            Func<Task> act = async () => await memberFeeService.AddAsync(new MemberFeeDto()
+            { FeeDate = new DateTime(2020, 2, 20), FeeAmount = 500 });
+
+            mockUnitOfWork.Verify(x => x.SaveChangesAsync(), Times.Never());
+            mockRepo.Verify(x => x.Add(It.IsAny<MemberFee>()), Times.Never());
+            act.Should().Throw<InvalidArgumentException>()
+                .WithMessage($"Invalid parameter: {nameof(MemberFeeDto.FeeDate)}, reason: {"Invalid FeeDate"}");
```

Comment:
I think you can just check if exception was thrown, without checking it's message

Reply:
Done